jobs/JRRFS_FCST

```diff
@@ -30,6 +30,8 @@ if [ -d ${DATA} ]; then # remove the ${DATA} directory if existed
   rm -rf ${DATA}
 fi
 mkdir -p ${DATA}
+
+export umbrella_data=${UNBRELLA_DATA:-${DATAROOT}}
```

Comment:
1. It is my impression that variables defined in a J-job and to be exported start with the uppercase letters.   I would suggest changing to:
`export UMBRELLA_DATA=${UMBRELLA_DATA:-${DATAROOT}}`

2. `UNBRELLA_DATA`  -> `UMBRELLA_DATA`

3. It looks like `umbrella_data` is a replicate of `DATAROOT` if not defined at the job card/workflow level. This may cause misunderstanding when users read the scripts.  
Suggest changing to:
`export UMBRELLA_DATA=${UMBRELLA_DATA:-UMBRELLA_DATA_not_defined}`
or
`export UMBRELLA_DATA=${UMBRELLA_DATA:-${DATAROOT}/umbrella}`

Reply:
Done